### PR TITLE
Implement ATACseq analysis

### PR DESCRIPTION
**Pull request by scvannost**
[CIDC-1069](https://dfcijira.dfci.harvard.edu:8443/browse/CIDC-1069)

### CODE REVIEW

File: cidc_schemas/schemas/assays/atacseq_analysis.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+	"$schema": "metaschema/strict_meta_schema.json#",
+	"$id": "atacseq_analysis",
+	"title": "Assay for Assay of Transposase-Accessible Chromatin by Sequencing",
```
Comment (roshni-b) [inline cidc_schemas/schemas/assays/atacseq_analysis.json:4]:
```suggestion
	"title": "Assay for Transposase-Accessible Chromatin by Sequencing",
```

File: cidc_schemas/template.py [hunk 2]
```diff
@@ -56,7 +60,9 @@
 
 def generate_analysis_template_schemas(
     target_dir: str = os.path.join(TEMPLATE_DIR, "analyses"),
-    fname_format: Callable[[str], str] = lambda file: f"{file}_analysis_template.json",
+    fname_format: Callable[
+        [str], str
+    ] = lambda file: f"{'rna_level1' if file == 'rna' else file}_analysis_template.json",
```
Comment (roshni-b) [inline cidc_schemas/template.py:65]: (This naming change can be a separate ticket in the backlog. We don't have multiple levels of RNA anymore, it's in a single `rna` assay/analysis going forward.)

**Review by roshni-b — APPROVED** [review]
Looks good!

File: cidc_schemas/template.py (hunk 2 above)
Comment (scvannost) [reply cidc_schemas/template.py:65]: should I revert this for this PR then?

File: cidc_schemas/schemas/assays/atacseq_analysis.json (hunk 1 above)
Comment (scvannost) [reply cidc_schemas/schemas/assays/atacseq_analysis.json]: updated to `Analysis for Assay of` the first A in ATACseq is "assay" interestingly enough

Comment (roshni-b) [reply cidc_schemas/schemas/assays/atacseq_analysis.json]: Ohh

**Pull request merged**